docs/src/pages/index.mdx

```diff
@@ -56,18 +28,35 @@ const Example: React.FC = () => {
 }
 
 export default Example
-```
 
-```typescript codesandbox=default
+---
+
+/*
+ * For now, the description can live in a code block.
+ * 
+ * If we don't put this in markdown, we will be authoring in .tsx instead of .mdx.
+ * 
+ * Later, we can update code-block.tsx to extract these, parse markdown and display as html
+ */
```

Comment:
maybe we write in frontmatter style and parse it out with something like https://github.com/docs/frontmatter